### DISCUSSION TITLE
Storage template issue

### DISCUSSION THREAD

**Discussion by mrbrahman** [immich-app/immich#10600] [Feature Request]
### The bug

Different people have different preferences for storing their photos in folders. Mine is:

`YYYY/YYYY-MM-DD <Optional Album Name>`

This helps me identify "albums" when casually browsing through the pics say on OS file explorer. This also allows for multiple albums on the same day. For me a pic belongs to only one album. If multiple things are needed I use tags (keywords).

It's great that Immich [storage template](https://immich.app/docs/administration/storage-template/) is designed to address the different storage templates! However, it does not seem to be working in this case.

My template:

`{{y}}/{{y}}-{{MM}}-{{dd}} {{{album}}}/{{filename}}`

This is resulting in a "." at the end of the folder name, for the folders that don't have album names (I don't think album names can ever be made mandatory for all folders).

![unnamed](https://github.com/immich-app/immich/assets/16898939/7e76b253-be14-4998-820f-e9de73ab6ccf)

### The OS that Immich Server is running on

Linux Mint 20.3 (Una)

### Version of Immich Server

v1.106.4

### Version of Immich Mobile App

v.106.3 build.143

### Platform with the issue

- [X] Server
- [ ] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
... standard (no change)
```

### Your .env content

```Shell
... standard (no change)
```

### Reproduction steps

```bash
1. set up the storage template as above
2. run the storage migration job
```

**Comment by mmomjian** [comment]
Immich uses . As the character when an asset is not in an album. This is designed so that you can sort non album photos into the current directory. This would be a feature request to have this changed / editable.

**Reply by mrbrahman** [reply]
Thanks for quickly looking into this!

**Comment by bo0tzz** [comment]
#6810
